pyiron_atomistics/atomistics/job/interactive.py

```diff
@@ -441,32 +445,34 @@ def _key_from_property(self, key, prop):
                 return_lst = prop(self).tolist() + return_lst
         return np.array(return_lst)
 
-    def _lst_from_property(self, key, prop=None):
+    def _lst_from_property(self, key):
         """
+        Fetch latest values for the given property.
+
+        Values are first looked up in the interactive cache, then in the interactive group in the HDF5 file
+        ('output/interactive/<key>') and if not found there via approriate getter from our super class (GenericOutput).
+        That means that key needs to be a property defined there.
 
         Args:
-            key (str):
-            prop (function):
+            key (str): output key
 
         Returns:
-
+            :class:`numpy.ndarray`: collected values from all previous steps
         """
-        return_lst = self._lst_from_cache(key)
-        hdf5_output = self._key_from_hdf(key)
-        if hdf5_output is not None and len(hdf5_output) != 0:
-            if isinstance(hdf5_output[-1], list):
-                return_lst = [np.array(out) for out in hdf5_output] + return_lst
-            else:
-                return_lst = hdf5_output.tolist() + return_lst
-        elif prop is not None:
-            prop_result = prop(self)
-            if prop_result is not None:
-                return_lst = prop_result.tolist() + return_lst
-        return np.array(return_lst)
+        cached = self._lst_from_cache(key)
+        fetched = self._key_from_hdf(key)
+        if fetched is None or len(fetched) == 0:
+            fetched = getattr(super(), key)
```

Comment:
Loading from `output/generic` should be handled here, but I'll be honest and say that I didn't do testing beyond my sample notebook.